packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.styles.scss

```diff
@@ -11,4 +11,8 @@
   .oui-datagrid-responsive {
     overflow: inherit;
   }
+
+  oui-datagrid.hide-pagination oui-pagination {
+    display: none !important;
+  }
```

Comment:
**question:** Can we try and use `ng-if` instead of adding a new class?

Reply:
I agree this would be better but is this possible when using oui-datagrid as a component ?

Reply:
I did not see any option to hide the pagination in `data-grid`.